rte_mis.install

```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
   }
 
 }
+
+/**
+ * Impelments hook_schema().
+ */
+function rte_mis_schema() {
+  $schema['lottery_school_seats_status'] = [
+    'description' => 'Stores the result of the lottery',
+    'fields' => [
+      'school_name' => [
+        'description' => 'School Name',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+      ],
+      'school_id' => [
+        'description' => 'School Id',
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+      ],
+    ],
+    'primary key' => ['school_id'],
+  ];
+
+  // Get the language from school config.
+  $school_config = \Drupal::config('rte_mis_school.settings');
+  $languages = $school_config->get('field_default_options.field_medium');
+
+  if ($languages) {
+    foreach ($languages as $key => $value) {
+      $column_name = $value . ' seats';
+      $schema['lottery_school_seats_status']['fields'][$column_name] = [
+        'description' => 'Total available seats for' . $key,
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+      ];
+    }
+  }
```

Comment:
- The column name should be the machine name of the language, not the label ie Hindi/English. Also, remove the "seats" from the column name.
- Change the description to Total RTE seat available for that language.

Reply:
sure